### ISSUE TITLE
Have Lepton Daisy description tell you at what percent(s) it activates

### ISSUE THREAD

**Issue by RandomNerd1** [xoxfaby/BetterUI#93]
**What feature would you like added to BetterUI?**

Instead of (or in addition to) simply saying how many times Lepton Daisy will go off, based on how many are held by everyone, it's probably a little more useful to know _when_ it will go off.

So for example if you have 3 of them, maybe the description could say it'll go off at "25%,50%,75%", something like that?

**Comment by Rafii2198** [comment]
I think the problem with that is on multiplayer this item stacks globally so it would show wrong data, it would refer to just your stacks. Unless there is a way to count Leptons from all alive players.

**Comment by xoxfaby** [comment]
Yes it will show from all people, this already exists for other items that stack globally ( gold boss seed guy item )

**Comment by Rafii2198** [comment]
That's cool, I didn't know that :smiley:  I would suggest that instead of showing a list as you suggested *("25%,50%,75%")* it should say something like "Activates every X%", that way it wouldn't create very long text lines with multiple stacks as well as it would be more readable (just one value vs multiple).

**Comment by xoxfaby** [comment]
Did I remember to implement this with ItemStats?
